content/project/board-election-process.adoc

```diff
@@ -25,178 +25,165 @@ A board comprised fully of write-in candidates runs the risk of being overweighe
 . Make sure that the board reflects the balance of different viewpoints — the needs of users; of committers; of organizations, both large and small; of commercial interests.
```

Comment:
Repetition of the line 17, is it wanted?

Reply:
Ah I see what you're saying there. I hadn't thought twice about it being weird. I've updated the second instance to make it a bit different, while still keeping the spirit of the message (I think).
```suggestion
. Make sure that the board reflects the balance of different viewpoints and needs of users, committers, organizations of any size, and commercial interests.
```

Reply:
Nice!